### PR TITLE
Fix type-o

### PR DESCRIPTION
**Pull request by sveno1990**
Fix type-o

Version(s):
4.15+

Link to docs preview:
Installation configuration parameters for Nutanix -> Additional Nutanix configuration parameters -> [ `platform:
  nutanix: defaultMachinePlatform: failureDomains:` table cell](https://80760--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_nutanix/installation-config-parameters-nutanix#installation-configuration-parameters-additional-nutanix_installation-config-parameters-nutanix).

QE review:
- [x] QE has approved this change.

### CODE REVIEW

File: modules/installation-configuration-parameters.adoc [hunk 1]
```diff
@@ -3235,7 +3235,7 @@ The name of one or more failures domains.
 
 |platform:
   nutanix:
-    defaulatMachinePlatform:
+    defaultMachinePlatform:
```
Comment (mburke5678) [inline modules/installation-configuration-parameters.adoc:3238]: @yanhua121 @tuxtof  @sgaoshang You helped review the [initial PR to add this code](https://github.com/openshift/openshift-docs/pull/69618) to the docs. Are you able to take a look at this change? It seems reasonable to me.

**Review by tuxtof — APPROVED** [review]

**Pull request merged**